consensus/istanbul/backend/backend_test.go

```diff
@@ -0,0 +1,352 @@
+// Copyright 2019 The klaytn Authors
+// This file is part of the klaytn library.
+//
+// The klaytn library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The klaytn library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
+
+package backend
+
+import (
+	"github.com/hashicorp/golang-lru"
+	"github.com/klaytn/klaytn/blockchain/types"
+	"github.com/klaytn/klaytn/common"
+	"github.com/klaytn/klaytn/consensus/istanbul"
+	istanbulCore "github.com/klaytn/klaytn/consensus/istanbul/core"
+	"github.com/klaytn/klaytn/consensus/istanbul/validator"
+	"github.com/klaytn/klaytn/crypto"
+	"github.com/klaytn/klaytn/event"
+	"github.com/klaytn/klaytn/governance"
+	"github.com/klaytn/klaytn/networks/p2p"
+	"github.com/klaytn/klaytn/params"
+	"github.com/klaytn/klaytn/reward"
+	"github.com/klaytn/klaytn/storage/database"
+	"math/big"
+	"testing"
+)
+
+var (
+	// testing node's private key
+	PRIVKEY = "ce7671a2880493dfb8d04218707a16b1532dfcac97f0289d770a919d5ff7b068"
+	// testing node should be in a committee in these blocks
+	committeeBlocks = []int64{0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22}
```

Comment:
Is this array necessary? I think we can simply set to `22` and use it for iteration.

Reply:
@jeonghonah You are right at the moment. But if we change  some logic in the future, the array may not be so linear as shown now. So I want the testing code itself not to be changed.